net/tac_plus/Makefile

```diff
@@ -0,0 +1,48 @@
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=tac_plus
+PKG_VERSION:=4.0.4.28
+PKG_REV:=1349a9eca2ded877e0d2ec1ebf89b8f18fe4088a
+PKG_RELEASE:=1
+
+PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION)-$(PKG_REV).tar.gz
```

Comment:
Please use tar.xz instead if you don't go the codeload route

Reply:
Switched to codeload.github.com as requested.